From 77924f7df1b940a3180eb253bca9a39bd9276554 Mon Sep 17 00:00:00 2001
From: razorloves <razorloves@gmail.com>
Date: Thu, 3 May 2018 22:22:53 -0500
Subject: [PATCH 154/249] wiki: Fix if statements on unmaintained devices

We still put [] when there's no maintainers, so it was causing the
condition to be true for all unmaintained devices.

Change-Id: Ib8e6aa5b7bb4ff01ade3d27513487559dd0653e3
---
 _includes/templates/device_build.md   | 2 +-
 _includes/templates/device_install.md | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/_includes/templates/device_build.md b/_includes/templates/device_build.md
index b29b926..d70697f 100644
--- a/_includes/templates/device_build.md
+++ b/_includes/templates/device_build.md
@@ -139,7 +139,7 @@ Then, run `source ~/.profile` to update your environment.
 
 ### Initialize the LineageOS source repository
 
-{% if device.maintainers %}
+{% if device.maintainers != empty %}
 The following branches are officially supported for the {{ device.vendor }} {{ device.name }}:
 {% else %}
 The following branches can be used to build for the {{ device.vendor }} {{ device.name }}:
diff --git a/_includes/templates/device_install.md b/_includes/templates/device_install.md
index 8f8d225..d66698c 100644
--- a/_includes/templates/device_install.md
+++ b/_includes/templates/device_install.md
@@ -41,7 +41,7 @@ There are no recovery install instructions for this discontinued device.
 {%- endif -%}
 {%- endcapture -%}
 
-{% if device.maintainers %}
+{% if device.maintainers != empty %}
 1. Download the [LineageOS install package](https://download.lineageos.org/{{ device.codename }}) that you'd like to install or [build]({{ "devices/" | append: device.codename | append: "/build" | relative_url }}) the package yourself.
 {% else %}
 1. [Build]({{ "devices/" | append: device.codename | append: "/build" | relative_url }}) a LineageOS install package.
-- 
2.17.1

